README.md

```diff
@@ -1,9 +1,1444 @@
throughput and how you manage capacity. This setting can be changed later.</p> <p>Billing Mode has two values, by default it will be set to <strong>PROVISIONED</strong>.</p> <ul> <li>PROVISIONED: Sets the billing mode to Provisioned Mode</li> <li>PAY PER REQUEST: Sets the billing mode to On-Demand Mode</li> </ul> <p> When the billing mode is Provisioned, read and write capacity becomes required parameters. The values for read and write capacity units are default set to 5. Also, auto-scaling is disabled. </p> </div> </li> <li> <strong> Secondary Indexes </strong> <div> <p> For creating Local Secondary Index (LSI) and Global Secondary Index (GSI), please refer the documentation above. </p> <p> Once table is created local secondary index cannot be added/updated, but Global Secondary can be created. </p> </div> </li> <li> <strong> Enable Stream </strong> <div> <p> A DynamoDB stream is an ordered flow of information about changes to items in a DynamoDB table. When you enable a stream on a table, DynamoDB captures information about every modification to data items in the table. To enable streams for table, set the enable_stream to true. </p> <p>If stream are enabled you need to select stream view type. There are four options you could select from: </p> <ul> <li> KEYS ONLY: Only the key attributes of the modified item are written to the stream. </li> <li> NEW IMAGE: The entire item, as it appears after it was modified, is written to the stream. </li> <li> OLD IMAGE: The entire item, as it appeared before it was modified, is written to the stream. </li> <li> NEW AND OLD IMAGES: Both the new and the old item images of the item are written to the stream. </li> </ul> </div> </li> <li> <strong>SSE</strong> <div> <p> To use the SSE set the SSE to true. </p> <p>If enabled (true), server-side encryption type is set to KMS and an Amazon Web Services managed key is used (KMS charges apply). If disabled (false) or not specified, server-side encryption is set to Amazon Web Services owned key.</p><p>If SSE is enable and KMS master key is provided, then that key will be used for encryption.</p> </div> </li> <li> <strong>KMS Master Key</strong> <p> The KMS key that should be used for the KMS encryption. To specify a key, use its key ID, Amazon Resource Name (ARN), alias name, or alias ARN. Note that you should only provide this parameter if the key is different from the default DynamoDB key alias/aws/dynamodb </p> </li> <li> <strong>Tags</strong> <div> <p> To add tags to you table, follow the step given below </p> <ul> <li> For a single values for tags, pass a JSON object. The JSON object should contain the "Key" and "Value" keys. These two keys are required and without these adding tags won't be possible. <p> { "Key" : "key_data", "Value" : "value_data" } </p> </li> <li> For multiple values in tags, pass a list of JSON object. <p> [ { "Key" : "key_data1", "Value" : "value_data1" }, { "Key" : "key_data2", "Value" : "value_data2" } ] </p> </li> </ul> </div> </li> </ul>
```

Comment:
```suggestion
<ul> <li> <strong> Partition and Sort Key </strong> <div> <p> Each table has a Primary Key to identify items uniquely. The Primary Key in DynamoDB is made up of a Partition Key and a Sort Key. The Sort Key is optional, but once the table is created you cannot add/update the Sort Key. </p> <p> All Partition and Sort Keys in Dynamodb have only three supported datatypes: String, Number and Binary</p> </div> </li> <li> <strong> Billing Mode </strong> <div> <p>The Billing mode defines how you will be charged for read and write throughput and how capacity is managed. This setting can be changed later.</p> <p>Billing Mode has two values, by default it will be set to <strong>PROVISIONED</strong>.</p> <ul> <li>PROVISIONED: Sets the billing mode to Provisioned Mode</li> <li>PAY PER REQUEST: Sets the billing mode to On-Demand Mode</li> </ul> <p> When the billing mode is Provisioned, read and write capacity become required parameters. The values for read and write capacity units are set to 5 by default. Also, auto-scaling is disabled. </p> </div> </li> <li> <strong> Secondary Indexes </strong> <div> <p> For creating Local Secondary Indexes (LSI) and Global Secondary Indexes (GSI), please refer the documentation above. </p> <p> Once a table is created, Local Secondary Indexes cannot be added/updated, but Global Secondary Indexes can be created. </p> </div> </li> <li> <strong> Enable Stream </strong> <div> <p> A DynamoDB stream is an ordered flow of information about changes to items in a DynamoDB table. When you enable a stream on a table, DynamoDB captures information about every modification to data items in the table. To enable streams for table, set enable_stream to true. </p> <p>If streams are enabled you need to select a stream view type. There are four options you could select from: </p> <ul> <li> KEYS ONLY: Only the key attributes of the modified item are written to the stream. </li> <li> NEW IMAGE: The entire item, as it appears after it was modified, is written to the stream. </li> <li> OLD IMAGE: The entire item, as it appeared before it was modified, is written to the stream. </li> <li> NEW AND OLD IMAGES: Both the new and the old item images of the item are written to the stream. </li> </ul> </div> </li> <li> <strong>SSE</strong> <div> <p> To use the SSE set the SSE to true. </p> <p>If enabled (true), server-side encryption type is set to KMS and an Amazon Web Services managed key is used (KMS charges apply). If disabled (false) or not specified, server-side encryption is set to Amazon Web Services owned key.</p><p>If SSE is enable and KMS master key is provided, then that key will be used for encryption.</p> </div> </li> <li> <strong>KMS Master Key</strong> <p> The KMS key that should be used for the KMS encryption. To specify a key, use its key ID, Amazon Resource Name (ARN), alias name, or alias ARN. Note that you should only provide this parameter if the key is different from the default DynamoDB key alias/aws/dynamodb </p> </li> <li> <strong>Tags</strong> <div> <p> To add tags to your table, follow the steps given below: </p> <ul> <li> For single-value tags, pass a JSON object. The JSON object should contain the "Key" and "Value" keys, which are both required. <p> { "Key" : "key_data", "Value" : "value_data" } </p> </li> <li> For multiple values in tags, pass a list of JSON object. <p> [ { "Key" : "key_data1", "Value" : "value_data1" }, { "Key" : "key_data2", "Value" : "value_data2" } ] </p> </li> </ul> </div> </li> </ul>
```

Reply:
Updated the code and resolved the issue.